src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
+  "Star Anise Pod": [],
+  "Ginger": [],
+  "Whole Cloves": [],
+  "Cardamom Pod": [],
+  "Vanilla Bean": [],
+  "Maple": [],
+  "Pear": [],
+  "Allspice": [],
+  "Apple Bitters": [],
+  "Amontillado Sherry": [],
```

Comment:
I think this should have an alias "sherry" unless you don't want the recipe to show unless they type in exactly "Amontillado Sherry"

Reply:
It looks like you know more than me here. Again, I don't want to force search aliases that don't belong, but if we can make it easier to find drinks given what people may search, we just really think on those because it's the point of the app. So maybe "Sherry" as an alias - and / or "Amontillado". Both up to you.